src/NHSD.BuyingCatalogue.Ordering.Api/Services/CompleteOrder/CompleteOrderService.cs

```diff
@@ -47,13 +49,20 @@ public async Task<Result> CompleteAsync(CompleteOrderRequest request)
 
             if (order.FundingSourceOnlyGMS.GetValueOrDefault())
             {
-                await using var stream = new MemoryStream();
-                await _createPurchasingDocumentService.CreatePatientNumbersCsvAsync(stream, order);
-                stream.Position = 0;
+                var patientNumbers = order.OrderItems.Where(x =>
+                    ProvisioningType.Patient.Equals(x.ProvisioningType) &&
```

Comment:
can these be inverted? To be like x.ProvisioningType.Equals(ProvisioningType.Patient)? It'd be easier to understand if they were